mmcv/runner/hooks/optimizer.py

```diff
@@ -34,6 +34,27 @@ def after_train_iter(self, runner):
         runner.optimizer.step()
 
 
+@HOOKS.register_module()
+class GradientCumulativeOptimizerHook(OptimizerHook):
+
+    def __init__(self, grad_clip=None, cumulative_iters=1):
+        super(GradientCumulativeOptimizerHook, self).__init__(grad_clip)
+        self.cumulative_iters = cumulative_iters
+
+    def after_train_iter(self, runner):
+        runner.outputs['loss'] = runner.outputs['loss'] / self.cumulative_iters
```

Comment:
Gradients will be scaled improperly with the current approach if the `runner.max_iters` is not divisible by `cumulative_iters`